bin/internal/canvaskit.version

```diff
@@ -1 +1,2 @@
-NcwvqeeKK7urddCbEdDvHytdaCiCA_8-4oS_T_ouGfgC
+8MSYGWVWzrTJIoVL00ZquruZs-weuwLBy1kt1AawJiIC
+
```

Comment:
I'm not sure if the Flutter tool likes empty lines in the `.version` files. Probably fine, just not sure.

Reply:
You are correct, fix pushed